### PR TITLE
[ENG-3811] Warning for special chars in help modal

### PR DESCRIPTION
**Pull request by futa-ikeda**
-   Ticket: [ENG-3811]
-   Feature flag: n/a

## Purpose
- add some text to warn users about special characters in the file browser's help modal

## Summary of Changes
- Update translation for Upload and Rename section of help modal

## Screenshot(s)
Screen Shot 2022-06-17 at 10 58 01 AM

## QA Notes

[ENG-3811]: https://openscience.atlassian.net/browse/ENG-3811?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by brianjgeiger — APPROVED** [review]

**Review by adlius — APPROVED** [review]

File: translations/en-us.yml [hunk 1]
```diff
@@ -1921,12 +1921,12 @@ osf-components:
             view_files_detail: 'Click a file name to go to view the file in the OSF. Opens file in a new tab.'
             upload: Upload
             upload_detail_first: 'There are two ways to upload files. Open the storage provider or folder where you intend to upload files; you can then drag files from your desktop into files list area OR click the green plus symbol' 
-            upload_detail_second: ', then click “Upload Files”, and select your files.'
+            upload_detail_second: ', then click “Upload Files”, and select your files. (Note: File names with special characters, like <code>&</code>, <code>%</code>, and <code>/</code> may cause unexpected behavior).'
```
Comment (chth0n1x) [inline translations/en-us.yml]: I would suggest not goading users who may be so inclined. More neutral phrasing or simple guidance not to use those characters is better. 'Note: Using file names with ...are not recommended.' may work. Also using 'such as' instead of 'like' is more professional. I would also remove the parenthesis as they are redundant with 'Note:'.

File: translations/en-us.yml [hunk 2]
```diff
@@ -1921,12 +1921,12 @@ osf-components:
...
             create_folder: 'Create a folder'
-            create_folder_detail: 'To create a folder to help organize your files, click the green plus symbol, then click “Create a Folder” (note: You can also organize your project content by creating Components).'
+            create_folder_detail: 'To create a folder to help organize your files, click the green plus symbol, then click “Create a Folder”. (Note: You can also organize your project content by creating Components).'
             rename: 'Rename a folder or file'
             rename_detail_first: 'In the files list, select the file that you want to rename and click the vertical ellipses'
-            rename_detail_second: 'to open the dropdown. Select “Rename”.'
+            rename_detail_second: 'to open the dropdown. Select “Rename”. (Note: Some special characters, like <code>&</code>, <code>%</code>, and <code>/</code> may cause unexpected behavior).'
```
Comment (chth0n1x) [inline translations/en-us.yml]: Same here regarding phrasing.

**Review by chth0n1x — CHANGES_REQUESTED** [review]
Hey Futa, these look good but at a minimum I would rephrase the 'like' -> 'such as' and remove the parenthesis.
